src/core/display/Container.js

```diff
@@ -606,17 +606,23 @@ Container.prototype.renderCanvas = function (renderer)
 
 /**
  * Destroys the container
- * @param [destroyChildren=false] {boolean} if set to true, all the children will have their destroy method called as well
+ * @param [opts] {object} Options parameter
+ * @param [opts.children=false] {boolean} if set to true, all the children will have their destroy
+ *      method called as well. 'opts' will be passed on to those calls.
+ *
  */
-Container.prototype.destroy = function (destroyChildren)
+Container.prototype.destroy = function (opts)
 {
+    opts = opts || {};
```

Comment:
@bigtimebuddy I think both your option would be breaking behaviour (Sprite and Text never destroyed children), I was more leaning towards ```{children: false, texture: <passedInBool1>, baseTexture: <passedInBool2>}``` for ```Sprite.destroy``` and ```{children: false, texture: <passedInBool1>, baseTexture: false}``` for ```Text.destroy```. Happy to do so if that's what you meant.

What are your thoughts on the deprecation warning?